vmdb/app/assets/javascripts/directives/miq_calendar.js

```diff
@@ -0,0 +1,37 @@
+miqAngularApplication.directive('miqCalendar', function() {
+  return {
+    require: 'ngModel',
+    link: function(scope, elem, attr, ctrl) {
+      console.log('miqCalendar', scope, elem, attr, ctrl);
+
+      // prevents clash with old calendar code
+      if (elem.attr('id').match(/^miq_angular_date|^miq_date/)) {
+        console.error("Can't use miqCalendar together with miqBuildCalendar magic, sorry", elem);
+        return;
+      }
+
+      // temporary - dhtmlx needs id
+      if (! elem.attr('id')) {
+        elem.uniqueId()
```

Comment:
this depends on jQuery UI - hoping we get rid of dhtmlxCalendar sooner than of jQuery UI